frontend/app/src/components/history/consts.ts

```diff
@@ -181,6 +182,13 @@ export const tradeLocations: TradeLocationData[] = [
     imageIcon: true,
     exchange: true
   },
+  {
+    identifier: EXCHANGE_SHAPESHIFT,
+    name: 'ShapeShift',
+    icon: require('@/assets/images/shapeshift-fox.svg'),
```

Comment:
Is this different than the shapeshit image we have for the fox token?

Reply:
It's pulled from https://drive.google.com/file/d/1NgnWwq78OY323FwxoYQbZj4BQZqVLk-L/view and I just verified it is identical to `frontend/app/src/assets/images/airdrops/FOX_Icon_dark.svg`

Reply:
I reverted to the FOX_Icon_dark.svg in this instance.